docs/control-structures.rst

```diff
@@ -28,6 +28,9 @@ something like::
         }
     }
 
+...note::
+    A function cannot accept a multi-dimensional array as an inpur parameter.
```

Comment:
Typo: `inpur` - and we could also add that it is possible if you enable the experimental ABIEncoderV2 feature.

Reply:
@chriseth Cool, will add this and add a new issue to describe how to do that more clearly that what exists now.